components/external-link/index.js

```diff
@@ -0,0 +1,28 @@
+/**
+ * External dependencies
+ */
+import classnames from 'classnames';
+
+/**
+ * WordPress dependencies
+ */
+import { __ } from 'i18n';
+
+/**
+ * Internal dependencies
+ */
+import Dashicon from '../dashicon';
+import './style.scss';
+
+function ExternalLink( { href, children, className, ...additionalProps } ) {
```

Comment:
I intentionally passed that through the href={ href } because without it the ExternalLink becomes useless (opens up a new empty tab). I know it's not a _real_ required parameter, but it felt better this way.